starwars/src/App.js

```diff
@@ -27,6 +30,7 @@ class App extends Component {
     return (
       <div className="App">
         <h1 className="Header">React Wars</h1>
+        {this.state.map((data, i) => <Child key={i} postData={data.results} />)}
```

Comment:
You need to map over the array in state, i.e. `this.state.data.results`, judging by how you wrote you state object.